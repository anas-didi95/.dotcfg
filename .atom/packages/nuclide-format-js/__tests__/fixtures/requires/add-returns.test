function one() {
  return two;
}

function three() {
  return {four};
}

var five = () => ({six});
