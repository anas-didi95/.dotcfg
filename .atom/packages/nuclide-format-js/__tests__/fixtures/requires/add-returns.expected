const four = require('four');
const six = require('six');
const two = require('two');

function one() {
  return two;
}

function three() {
  return {four};
}

var five = () => ({six});
